examples/worldview/src/features/stage/StageMap.js

```diff
@@ -54,7 +55,7 @@ export class StageMap extends Component {
 
   _resizeVideo() {
     const {width, dimension} = this.props;
-    this._setDevicePixelRatio(dimension.width / width);
+    this._setDevicePixelRatio(nearestEvenFloat(dimension.width / width));
```

Comment:
what's the reason behind using Float here and Int above?

Reply:
The ratio is a scaling factor and uses float (e.g. 1.345), whereas the dimensions represent pixels and those should be int (e.g. 1080 or 720).